PC02_Graffiti_COOPER-SANSONE.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Andrew Cooper-Sansone (9/4/2020)
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
+#=======Add your code here======
+#To Jeff Bezos' picture, add at least 2 different geometric shapes - 
+# this could be a circle, a polygon (+4 sides), and/or a line.
+
+#FOR FULL CREDIT, these shapes must:
+#1. have different weights (or thicknesses) of the lines
+#2. be drawn at different positions on the screen (you can use goto(x,y) for a specific position, or pick up the pen up() and use forward() and turns (left(), right()) to navigate your turtle)
+
+#EXTRA CREDIT
+#0.5 pts - create a 3rd shape
+#0.5 pts - set at least one color using RGB values.
+#0.5 pts - create one unfilled and one filled shape
+
+Turtle()
+clear()
+up()
+goto(110, -90)
+width(1)
+down()
+color(100, 100, 100)
+rt(90)
+circle(10)
+circle(15)
+circle(20)
+circle(25)
+circle(30)
+circle(35)
+circle(40)
+circle(45)
+circle(50)
+circle(55)
+circle(60)
+circle(65)
+circle(70)
+circle(75)
+circle(80)
+circle(85)
+circle(90)
+circle(95)
+circle(100)
+rt(90)
+circle(10)
+circle(15)
+circle(20)
+circle(25)
+circle(30)
+circle(35)
+circle(40)
+circle(45)
+circle(50)
+circle(55)
+circle(60)
+circle(65)
+circle(70)
+circle(75)
+circle(80)
+circle(85)
+circle(90)
+circle(95)
+circle(100)
+rt(90)
+circle(10)
+circle(15)
+circle(20)
+circle(25)
+circle(30)
+circle(35)
+circle(40)
+circle(45)
+circle(50)
+circle(55)
+circle(60)
+circle(65)
+circle(70)
+circle(75)
+circle(80)
+circle(85)
+circle(90)
+circle(95)
+circle(100)
+rt(90)
+circle(10)
+circle(15)
+circle(20)
+circle(25)
+circle(30)
+circle(35)
+circle(40)
+circle(45)
+circle(50)
+circle(55)
+circle(60)
+circle(65)
+circle(70)
+circle(75)
+circle(80)
+circle(85)
+circle(90)
+circle(95)
+circle(100)
+up()
+goto(-40, 240)
+dot(30, "blue")
+up()
+goto(-45, 255)
+dot(15, "blue")
+up()
+goto(-50, 262.5)
+dot(7.5, "blue")
+up()
+goto(-55, 266.5)
+dot(3.75, "blue")
+up()
+width(2)
+color(220,20,60)
+goto(40, 120)
+down()
+goto(750, 100)
+up()
+goto(-15, 110)
+down()
+goto(750, 100)
```

Comment:
After setting the coordinates corresponding to Jeff's eyes, as well as the point at which the laser beams are shooting, as variables (e.g. left_eye = -15, 110; laserTarget = 750, 100) (in the Defining Variables section above), replace the values in the parentheses with corresponding variable names.
```suggestion
goto(right_eye)
down()
goto(laserTarget)
up()
goto(left_eye)
down()
goto(laserTarget)

```